File: C/Vetor/media pares.c
#include<stdio.h>
int main(){
int i, n;
double media=0, y=0;
    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d", &n);
        int vet[n];

for (i=0; i<n; i++){
    printf("Digite um numero:");
    scanf("%d", &vet[i]);
        if(vet[i] % 2 == 0){
            media = media + vet[i];
            y = y + 1;
        }
}
    if (media==0){
        printf("NENHUM NUMERO PAR...");
    }else{
    media = media/y;
    printf("\nMEDIA DOS PARES = %.1lf", media);
    }



}

File: Phyton/Vetor/mediapares.py
n = int(input("Quantos elementos vai ter o vetor ?"))

vet = [0 for x in range(n)]
media = 0
cont = 0
for i in range(n):
    vet[i] = int(input("Digite um numero: "))
    if vet[i] % 2 == 0:
        media += vet[i]
        cont += 1

if media != 0:
    print(f"MEDIA DOS PARES = {media/cont:.1f}")
else:
    print("NENHUM NUMERO PAR")

